custom-recipes/nlp-analysis-ontology-tagging/recipe.py

```diff
@@ -15,21 +14,20 @@
     lang, lang_column = settings.language, ""
 
 tagger = Tagger(
-    settings.text_input,
-    settings.ontology_input,
+    settings.text_input.get_dataframe(),
```

Comment:
Nitpick, it's better to use named keyword arguments instead of relying on argument order. Simpler to debug in case you have changed the order of some parameters and something fails/doesn't behave.

A bit more verbose, but can save you a lot of debugging time

Reply:
you are definitely right ! here : d11d305